frontend/packages/console-shared/src/hooks/deep-compare-memoize.ts

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
 import * as _ from 'lodash';
 
-export const useDeepCompareMemoize = <T = any>(value: T, strinfigy?: boolean): T => {
+export const useDeepCompareMemoize = <T = any>(value: T, stringify?: boolean): T => {
```

Comment:
Maybe this is intentional. Also, it's a shared function among other teams (since it's in `console-shared`)
I think you should leave this as it does not affect ur change.